frontend/packages/kubevirt-plugin/src/components/vms/vm-details.tsx

```diff
@@ -106,6 +145,34 @@ export const VMDetails: React.FC<VMDetailsProps> = (props) => {
         {OSMismatchAlert}
         <HashAnchor hash="details" />
         <SectionHeading text={`${kindObj.label} Details`} />
+        {kindObj === VirtualMachineModel && isVMRequireRestart && (
+          <div>
+            <Alert
```

Comment:
I think the alert should be on top of the title, like in the upload pvc designs
@matthewcarleton can you approve?

Reply:
In the design the warning is below the title:
https://github.com/openshift/openshift-origin-design/pull/407/files?short_path=81b238a#diff-81b238a484a05d37b10818f4218999ec

Reply:
I wonder if we should actually change it to align with what is done for operators WDYT @yfrimanm ?
![image](https://user-images.githubusercontent.com/287880/86927336-98c2ef80-c109-11ea-8f12-a8296e830984.png)

Reply:
this would be better for changes that are covered across all tabs

Reply:
+1 @matthewcarleton we'll certainly want to follow this convention for changes that are relevant across all tabs with VMs as well.